adapter/ods-adapter-service.iml

```diff
@@ -0,0 +1,1136 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There is a comment in the gitignore of the adapter module that specifies why the intellij config files are not included in the gitignore. I usually ignore those files aswell in my projects but since i didn't write that module I left the gitignore how it was.
Code in question:
### Intellij Patch ###
# Comment Reason: https://github.com/joeblau/gitignore.io/issues/186#issuecomment-215987721

# *.iml
# modules.xml
# *.ipr

Reply:
Ah alright, wasn't aware. Thanks for clarification!